cypress/factories/bitcoin/account-xpub.ts

```diff
@@ -1,4 +1,6 @@
-import { BitcoinAccount } from '@shapeshiftoss/unchained-client/dist/generated/bitcoin'
+import { bitcoin } from '@shapeshiftoss/unchained-client'
+
+type BitcoinAccount = bitcoin.BitcoinAccount
```

Comment:
Would prefer either destructuring `BitcoinAccount` off `bitcoin` instead of type alias, or I don't think it is terrible to reference the type directly as `bitcoin.BitcoinAccount`